include/coruja/algorithm/remove_if.hpp

```diff
@@ -0,0 +1,38 @@
+
+// Copyright Ricardo Calheiros de Miranda Cosme 2018.
+// Distributed under the Boost Software License, Version 1.0.
+// (See accompanying file LICENSE or copy at
+//  http://www.boost.org/LICENSE_1_0.txt)
+
+#pragma once
+
+#include <algorithm>
+
+namespace coruja { namespace algorithm {
```

Comment:
I think the directory `algorithm` should be a child of `coruja/container` and not `coruja` because the goal is to have a group of algorithms related to containers and views. BTW, we shouldn't use here the rule that maps a directory to a namespace, in other words, I think it is better to remove the namespace `algorithm` because any function is potentially an algorithm. What do you think?